.github/workflows/test.yml

```diff
@@ -114,3 +123,85 @@ jobs:
           export PATH="$HOME/.pyenv/versions/$(cat .python-version)/bin:$PATH"
           if test -d .env/bin/; then source .env/bin/activate; else source .env/Scripts/activate; fi
           just test
+
+      - name: Build the wheels
+        shell: bash
+        run: |
+          export PATH="$HOME/.cargo/bin:$PATH"
+          export PATH="$HOME/.pyenv/versions/$(cat .python-version)/bin:$PATH"
+          if test -d .env/bin/; then source .env/bin/activate; else source .env/Scripts/activate; fi
+          maturin list-python
+          PYTHON_INTERPRETER=$(maturin list-python | grep -o "CPython $(cat .python-version | grep -o -E '^[^\.]+\.[^\.]+').* at .*" | cut -d' ' -f 4 | tr '\\' '/')
+          echo "Selected interpreter: ${PYTHON_INTERPRETER}"
+          just build-all-wheels "${PYTHON_INTERPRETER}" "${{ matrix.target.rust-target }}"
+
+      - name: Inspect wheels
+        shell: bash
+        run: |
+          ls -lhaGiR target/wheels/*.whl
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: wheels-${{ matrix.python }}-${{ matrix.target.id }}
+          path: target/wheels
+          if-no-files-found: error
+          retention-days: 1
+
+  prerelease:
+    name: Pre-Release
+
+    runs-on: ubuntu-18.04
+    if: startsWith(github.ref, 'refs/heads/prepare-')
```

Comment:
I wonder if we could trigger this with a keyword in a commit message. Like `[prerelease]`. Thoughts?

Reply:
I think we can but I also think it preferable to have the release preparation in a separate branch and force this.